packages/docusaurus-plugin-client-redirects/src/index.ts

```diff
@@ -43,3 +45,7 @@ export default function pluginClientRedirectsPages(
     },
   };
 }
+
+export function trimBaseUrl(path: string, baseUrl: string): string {
```

Comment:
we already have removeSuffix in docusaurus-utils, I'd suggest adding removePrefix in this package instead as this kind of function is often useful in multiple places